frontend/src/shared/helpers.tsx

```diff
@@ -145,64 +143,6 @@ export const RequestsListColumns: ColDef[] = [
   },
 ];
 
-export function prepareRequestDataForAgGrid(
-  requestsListQueryResult: RequestsListQuery
-) {
-  const newRequests = requestsListQueryResult.requests.map((request) => {
-    return {
-      totalSamples: request.hasSampleSamplesConnection?.totalCount,
-      ...request,
-    };
-  });
-
-  return {
-    requestsConnection: requestsListQueryResult.requestsConnection,
-    requests: newRequests,
-  };
-}
-
-export function preparePatientDataForAgGrid(
-  patientsListQueryResult: PatientsListQuery
-) {
-  const newPatients = patientsListQueryResult.patients.map((patient) => {
-    const samples = patient.hasSampleSamples;
-    const patientAliases = patient.patientAliasesIsAlias;
-
-    const cmoPatientId = patientAliases?.find(
-      (pa) => pa.namespace === "cmoId"
-    )?.value;
-    const dmpPatientId = patientAliases?.find(
-      (pa) => pa.namespace === "dmpId"
-    )?.value;
-
-    const cmoSampleIds = samples?.map((s) => {
-      const sampleMetadata = s.hasMetadataSampleMetadata[0];
-      return sampleMetadata?.cmoSampleName || sampleMetadata?.primaryId;
-    });
-
-    const additionalProperties =
-      samples[0]?.hasMetadataSampleMetadata[0]?.additionalProperties;
-    const additionalPropertiesJson = additionalProperties
-      ? JSON.parse(additionalProperties)
-      : {};
-
-    return {
-      cmoPatientId,
-      dmpPatientId,
-      cmoSampleIds,
-      smilePatientId: patient.smilePatientId,
-      totalSamples: patient.hasSampleSamplesConnection?.totalCount,
-      consentPartA: additionalPropertiesJson["consent-parta"],
-      consentPartC: additionalPropertiesJson["consent-partc"],
-    };
-  });
-
-  return {
-    patientsConnection: patientsListQueryResult.patientsConnection,
-    patients: newPatients,
-  };
-}
-
```

Comment:
Removed functions that flatten data on the frontend. This step is now done at the GraphQL server level.